students/ethan_nguyen/lesson03/mailroom.py

```diff
@@ -1,87 +1,76 @@
 
-import operator
-'''
-Class to store Donor object and its attributes
-'''
-class Donor:
-    def __init__(self, name, amount=0, num=0):
-        self.name = name
-        self.amount_gift = amount
-        self.num_gift = num
-    '''function to calculate average donation'''
-    def getAverge(self):
-        return (self.amount_gift/self.num_gift)
-    '''function to return number of donations'''
-    def getCount(self):
-        return self.num_gift
-    '''function to return historical amount'''
-    def getAmount(self):
-        return self.amount_gift
-    def getName(self):
-        return self.name
-    '''function to add new donation amount to historical'''
-    def addAmount(self, addition):
-        self.amount_gift += addition
-        self.num_gift += 1
+import sys
+
+#create a donor_db as a list
+donor_db = [("William Gates, III", [653772.32, 12.17]),
+            ("Jeff Bezos", [877.33]),
+            ("Paul Allen", [663.23, 43.87, 1.32]),
+            ("Mark Zuckerberg", [1663.23, 4300.87, 10432.0]),
+            ("Donald Trump", [50000.24, 1002])
+            ]
+
+prompt = "\n".join(("Welcome to the mail room",
+          "Please choose from below options:",
+          "S - Send a Thank You",
+          "C - Create a Report",
+          "Q - Exit",
+          ">>> "))
+
+
+def view_donors():
+    print("\n".join(donor_db))
+
+def quit_program():
+    print("Thank you. Bye!")
+    sys.exit(0)  # exit the interactive script
+
+def send_thank_you():
+
+    input_Name = input("Please enter a name > ")
+    while("list" in input_Name):
+        for d in donor_db:
+            print(d[0])
+        input_Name = input("Please enter a name > ")
```

Comment:
reminder that var names are snake_case (all lower case separated by underscores) -> `input_name`